integrations/envoy/envoy-monitoring/README.md

```diff
@@ -19,67 +19,105 @@ This integration includes the following resources:
+The Envoy Metrics integration supports disabling redirect warnings.
 
-   Example:
+To disable redirect warnings, install this integration.
+Then, update the `envoy-metrics` check to add the `--redirect-ok` flag in the check's `command` attribute.
 
-   ```yaml
-   spec:
-     command: >-
-       http-get
-       --timeout 10
-       --url "http://127.0.0.1:9901/stats/prometheus
-       --header "Content-Type: text/plain"
-       --header "X-Example-Header: helloworld"
-   ```
+**Example**:
+
+```yaml
+spec:
+ command: >-
+   http-get
+   --timeout 10
+   --redirect-ok
+   --url "http://127.0.0.1:9901/stats/prometheus"
+```
+
+### Custom request headers
+
+The Envoy Metrics integration supports custom request headers.
+
+To use custom request headers, install this integration.
+Then, update the `envoy-metrics` check to add one or more `--header` flags in the check's `command` attribute.
+
+**Example**:
+
+```yaml
+spec:
+  command: >-
+    http-get
+    --timeout 10
+    --url "http://127.0.0.1:9901/stats/prometheus"
+    --header "Content-Type: text/plain"
+    --header "X-Example-Header: helloworld"
+```
+
+### Token substitution
+
+The Envoy Metrics integration supports Sensu [tokens] for variable substitution with data from Sensu entities.
 
 ## Plugins
 
 <!-- Links to any Sensu Integration dependencies (i.e. Sensu Plugins) -->
 
+The Envoy Metrics integration uses the following Sensu [plugins]:
+
 - [sensu/http-checks][http-checks-bonsai] ([GitHub][http-checks-github])
 
-## Metrics & Alerts
+## Alerts
 
-<!-- List of all metrics or events collected by this integration. -->
+<!-- List of all alerts generated by this integration. -->
 
-This integration collects a wide range of [metrics]. For a full list, see the [Envoy metrics documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/stats) and the [Envoy metrics example output](https://gist.github.com/thoward/02617f95accdfd7694acc671709c61d0).
+The Envoy Metrics integration produces the following alerts.
 
-This integration configures three alerts based on the following metrics:
+**envoy_server_live** (Boolean)
 
-- The `envoy_server_live` metric. This metric is a boolean value that indicates if Envoy is in a `LIVE` state (`1`) or is not (`0`). If Envoy is not in the `LIVE` state, a `CRITICAL` status alert will be raised. 
-- The `envoy_server_state` metric. This metrics is a gauge value that indicates the current state of the server. If the value is anything other than `LIVE` state (`0`), a `CRITICAL` status alert will be raised.
-- The `envoy_server_days_until_first_cert_expiring` metric. This metric is a gauge indicating how many days until the first cert expires. If the value is 15 days or less, a `WARNING` status alert will be raised, and if it is seven (7) days or less, a `CRITICAL` status alert will be raised.
+Generates a CRITICAL event if the Envoy server is not in a `LIVE` state (`1`).
 
-# Reference Documentation
+**envoy_server_state** (gauge)
```

Comment:
Are the `envoy_server_live` and `envoy_server_state` alerts essentially the same?